src/main/resources/db/changesets/20240723_updatesnapshotbuilderrequestspecifications.yaml

```
@@ -0,0 +1,13 @@
+databaseChangeLog:
+  - changeSet:
+      id: clear_snapshot_builder_requests
+      author: srubenstein
+      # Delete all rows from the snapshot_request
+      # table due to breaking change to json structure
+      changes:
+        - update:
+            tableName: snapshot_request
+            columns:
+              - column:
+                  name: snapshot_specification
+                  value: "{\"cohorts\": [{\"name\": \"test\",\"criteriaGroups\": []}],\"outputTables\": [{\"name\": \"Drug\",\"columns\": []}]}"
```

This works but you could be even more minimal, if you wanted to. `name` is ignored by the service and can be `null`, and both arrays are empty if missing. So this should work too
```suggestion
                  value: "{\"cohorts\": [{}], \"outputTables\": [{\"name\": \"Drug\"}]}"
```